tests/basic-test.sh

```diff
@@ -437,7 +437,7 @@ echo "ok user checkout"
 $OSTREE commit ${COMMIT_ARGS} -b test2 -s "Another commit" --tree=ref=test2
 echo "ok commit from ref"
 
-$OSTREE commit ${COMMIT_ARGS} -b test2 -s "Another commit with modifier" --tree=ref=test2 --owner-uid=`id -u`
+$OSTREE commit ${COMMIT_ARGS} -b test2 -s "Another commit with modifier" --tree=ref=test2 --owner-uid=0
```

Comment:
I think we should instead stop running this test in `bare-user-only` mode, but continue to run it for other repo modes.  And while we're here, use e.g. `ostree ls` to verify that we got the expected uid in the commit.

Reply:
Reading through the whole testsuite, my understanding is that this basic test is only here to reach the modifier logic (and check that it doesn't blatantly segfaults, I think). To that extent, tweaking the numeric value here should play the same role.

There are already other tests doing the check you are looking for, e.g. https://github.com/ostreedev/ostree/blob/38c14b374567f736533f7f515595208fe359819f/tests/test-basic-root.sh#L40-L42